src/connection/host_lookup.c

```diff
@@ -36,40 +36,49 @@
 static pthread_mutex_t gethostbyname_lock = PTHREAD_MUTEX_INITIALIZER;
 #endif /* HAVE_GETHOSTBYNAME_R */
 
-
 #define HOSTNAME_BUF_SIZE              (256)
+#define MAX_HOSTS_LINE_NUM       (256)
+#define IPADDR_LEN              (17)
+
+typedef enum
+{
+  HOSTNAME_LOOKUP = 0,
+  IPADDRESS_LOOKUP = 1,
+} LOOKUP_STATUS;
 
 struct cub_hostent
 {
-  char addr[17];
+  char addr[IPADDR_LEN];
```

Comment:
I think

`char ipaddr[IPADDR_LEN];`

could be better

Reply:
Right. I would verify this array name. 
Thank you.